src/components/ErrorPage/ErrorPage.tsx

```diff
@@ -0,0 +1,23 @@
+import { CubesIcon } from '@patternfly/react-icons';
+import { Link } from 'react-router-dom';
+import { StateCard } from '../EmptyStates/StateCard';
+import { PageLayout } from '../PageLayout/PageLayout';
+import styles from './ErrorPage.module.css';
+
+interface IErrorPage {
+  pageTitle?: string;
```

Comment:
I would be strict also in this case, let's remove `?`, then also default values bellow will be removed